examples/next/docs/js/accessibility/package.json

```
@@ -0,0 +1,18 @@
+{
+  "name": "a11y-js-demo",
+  "private": true,
+  "version": "0.0.0",
+  "type": "module",
+  "scripts": {
+    "dev": "vite",
```

```suggestion
    "start": "vite",
```
I've checked other examples, and they all use the `npm run start` command. Can do the same here?